spec/controllers/media_controller_spec.rb

```diff
@@ -1,12 +1,19 @@
 require 'rails_helper'
 
 RSpec.describe AtomicCms::MediaController, type: :controller do
+  # Previously the engines guide contained an incorrect example that
+  # recommended using the `use_route` option to test an engine's controllers
+  # within the dummy application. That recommendation was incorrect and has
+  # since been corrected. Instead, you should override the `@routes` variable
+  # in the test case with `Foo::Engine.routes`.
+  before { @routes = AtomicCms::Engine.routes }
+
   it 'accepts a post request and fails with bad data' do
     scrubber = double('scrubber')
     allow(MediaScrubber).to receive(:new).and_return(scrubber)
     allow(scrubber).to receive(:save).and_return(false)
 
-    post :create, use_route: :atomic_cms, file: double('file')
+    post :create, file: double('file')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.